maven-failsafe-plugin/src/main/java/org/apache/maven/plugin/failsafe/VerifyMojo.java

```diff
@@ -139,6 +139,13 @@
     @Parameter( property = "failIfNoTests" )
     private Boolean failIfNoTests;
 
+    /**
+     * Set this to a value greater than 0 to fail the whole test set if the cumulative number of flakes reaches
+     * this threshold. Set to 0 to allow an unlimited number of flakes.
```

Comment:
based on the code a negative number would do the same, right?
Should we maybe rather fail in that case?

Reply:
@oehme
Actually yes, we should and there is something like "verifyParameters" method.
Have you used the flag to serialize the content in the method `getConfigChecksum`?